plugins/functions/string.c

```diff
@@ -294,6 +304,165 @@ func_string_tokenize(grn_ctx *ctx, int n_args, grn_obj **args,
   return tokens;
 }
 
+#ifdef GRN_WITH_ONIGMO
```

Comment:
`GRN_SUPPORT_REGEXP` is better:

```suggestion
#ifdef GRN_SUPPORT_REGEXP
```

There is no difference between `GRN_WITH_ONIGMO` and `GRN_SUPPORT_REGEXP` for now. But we may add more conditions to define `GRN_SUPPORT_REGEXP`.